proto/asgt/type/target_metrics.proto

```diff
@@ -18,3 +18,23 @@ message TargetMetrics {
   string target = 1;
   repeated Metric metrics = 2;
 }
+
+message TargetMetricsV2 {
+  message Metric {
+    float precision = 1;
+    float answer_rate = 2;
+    int32 true_positive = 3;
+    int32 true_negative = 4;
+    int32 false_positive = 5;
+    int32 false_negative = 6;
+  }
```

Comment:
Can't we just take the `message Metric` out of the `TargetMetrics` message, we could reuse it here 🙂 It does contain `confidence`, but it will just be unsued here.

Reply:
Well since I am make a complete new super message, I might  as well do it the right way so it is not confusing

Reply:
I will change dont worry